FinancialTransactionsApi/V1/UseCase/Interfaces/IGetByTargetIdsUseCase.cs

```diff
@@ -1,14 +1,12 @@
-using System;
-using System.Collections.Generic;
 using System.Threading.Tasks;
 using FinancialTransactionsApi.V1.Boundary.Request;
-using FinancialTransactionsApi.V1.Domain;
-using Hackney.Core.DynamoDb;
+using FinancialTransactionsApi.V1.Boundary.Response;
+using FinancialTransactionsApi.V1.Helpers.GeneralModels;
 
 namespace FinancialTransactionsApi.V1.UseCase.Interfaces
 {
     public interface IGetByTargetIdsUseCase
     {
-        Task<PagedResult<Transaction>> ExecuteAsync(TransactionByTargetIdsQuery targetIdsQuery);
+        Task<PaginatedResponse<TransactionResponse>> ExecuteAsync(TransactionByTargetIdsQuery targetIdsQuery);
```

Comment:
It would be great to see the removal of the old Paging classes as well - `PagedResult<T>` etc

Reply:
Done